go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/icinga/icinga-notifications
 
-go 1.21
+go 1.22
 
 require (
 	github.com/creasty/defaults v1.7.0
 	github.com/emersion/go-sasl v0.0.0-20231106173351-e73c9f7bad43
 	github.com/emersion/go-smtp v0.21.2
 	github.com/goccy/go-yaml v1.11.3
 	github.com/google/uuid v1.6.0
-	github.com/icinga/icingadb v1.1.1-0.20230418113126-7c4b947aad3a
+	github.com/icinga/icinga-go-library v0.1.1-0.20240529153148-5054d55f8005
```

Comment:
I just tagged the new version as `v0.2.0`, please update this to use the tagged version.